projects/docs/docs/commands/task.md

```diff
@@ -76,6 +76,7 @@ You might wonder what the return value of `Tuist.graph()` is - the method return
 | ---------- | ------------------------------------------------------------- | ------- |
 | `name`| Name of the project | `String` |
 | `path` | Absolute path of the project | `String` |
+| `isExternal` | Indicates whether the target is imported through `Dependencies.swift`. | | `Bool` |
```

Comment:
There is an empty column before `Bool` which should not be there afaict

Reply:
@laxmorek I suggest to format the table to align the column so it's easier to understand when looking at the source file

Reply:
Oh, of course. Sorry. I'm too tired. I will fix it.